CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 
 * Changed
   * Updated `uuid` dependency to `v8.0.0`.
+  * Lock down terra-navigation-prompt to ensure MVB of terra-notification-dialog is consumed correctly
```

Comment:
This looks like it's in a released section

Reply:
My local version was out of date. It was correct initially 😓  I swear!